app/controllers/regulatory-statements/edit.js

```diff
@@ -1,9 +1,7 @@
 import Controller from '@ember/controller';
```

Comment:
What is the reason for the import order change?

Reply:
I was tired of the mess of imports, with newlines all over the place, so just snapped and arranged it by external/editor/plugins...

Reply:
ok :smile: